.env

```diff
@@ -0,0 +1,3 @@
+REACT_APP_BASE_API_URL=https://nimble-survey-web-staging.herokuapp.com/
+REACT_APP_CLIENT_ID=vFUr2Tf506a3WARm91KhMmmaP_WBX7ok6KCaI44Labo
+REACT_APP_CLIENT_SECRET=UGsOHj8hr8uLHFFPrtM8G6XzTZ2YiPrvAwX0cRkrLiY
```

Comment:
I am not sure if the prefix `REACT_APP_` is really needed. It could be removed to keep things simpler or maybe shortened to `APP_`.